src/components/Leftbar/Character.vue

```diff
@@ -153,6 +153,11 @@ export default {
         this.addint = 0;
         return;
       }
+
+      // Validate the character data
+      if (this.statpoints < 0 || this.newlevel > 120) {
+        return;
```

Comment:
Instead of returning here, we can just force the values instead which might actually benefit user experience (if someone wants to just quickly dump all their stats into one stat for example, they can just type 1111 instead of having to exactly type in 238). 

Force the level to `Utils.maxLevel` (assuming it's defined as per the comment above) if `newlevel > Utils.maxLevel`, and if `statpoints < 0` then perhaps do some math to subtract stats from whichever stat they are attempting to add the most points into until `statpoints` is back at 0 (So if I enter 500 into STR, subtract the right amount from STR for `statpoints` to be just at 0. If I enter 100 into STR and 200 into DEX, choose the highest I'm attempting to add -- so DEX -- and subtract enough).

Reply:
I like the idea of quickly consuming the remaining stat points but I feel like that logic would be a bit confusing to the user. I added a button that consumes the remaining stat points or sets level to max. Let me know what you think.

Reply:
That's a good point, I like it!